src/plugin.c

```diff
@@ -242,9 +242,15 @@ static int register_callback (llist_t **list, /* {{{ */
 		old_cf = le->value;
 		le->value = cf;
 
-		WARNING ("plugin: register_callback: "
-				"a callback named `%s' already exists - "
-				"overwriting the old entry!", name);
+		if (
+			old_cf->cf_callback != cf->cf_callback ||
+			old_cf->cf_udata.data != cf->cf_udata.data
+		)
```

Comment:
We don't usually have linebreaks after the opening or before the closing parenthesis:

    if ((old_cf->cf_callback != cf->cf_callback) || (...))